packages/connect/src/implementation.ts

```diff
@@ -71,8 +72,10 @@ export interface HandlerContext {
    */
   readonly service: ServiceType;
 
-  // TODO
-  readonly deadline?: AbortSignal;
+  /**
+   * An AbortSignal that is aborted when the connection with the client is closed or when the deadline is reached.
+   */
```

Comment:
We wrap JSDoc at 80 columns:

```suggestion
  /**
   * An AbortSignal that is aborted when the connection with the client is closed 
   * or when the deadline is reached.
   */
```